assets/js/blocks/cart-checkout/checkout/block.js

```diff
@@ -333,9 +333,12 @@ const Block = ( {
 							{ attributes.showReturnToCart && (
 								<ReturnToCartButton
 									link={
-										getSetting(
-											'page-' + attributes.cartPageId
-										) || CART_URL
+										attributes.cartPageId
+											? getSetting(
+													'page-' +
+														attributes.cartPageId
+											  )
+											: false
```

Comment:
This could be simplified to:

```suggestion
										 getSetting(
													'page-' +
														attributes?.cartPageId,
													false
											  )
```

Reply:
Isn't this an unnecessary lookup into the settings array though?

Reply:
Not really expensive imo. The array is not large.